BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -282,31 +285,43 @@ extension AutoContributeDetailViewController: UITableViewDataSource, UITableView
         cell.selectionStyle = .none
       case .supportedSites:
         cell.label.text = Strings.AutoContributeSupportedSites
-        cell.accessoryLabel?.attributedText = totalSitesAttributedString(from: upcomingContributions.count)
+        cell.accessoryLabel?.attributedText = totalSitesAttributedString(from: publishers.count)
         cell.selectionStyle = .none
       case .excludedSites:
-        // FIXME: Use actual number
-        let numberOfExcludedSites = 5
+        let numberOfExcludedSites = ledger.numberOfExcludedPublishers
         cell.label.text = String(format: Strings.AutoContributeRestoreExcludedSites, numberOfExcludedSites)
         cell.label.textColor = Colors.blurple400
       }
       return cell
     case .contributions:
-      if upcomingContributions.isEmpty {
+      if publishers.isEmpty {
         let cell = tableView.dequeueReusableCell(for: indexPath) as EmptyTableCell
         cell.label.text = Strings.EmptyAutoContribution
         return cell
       }
-      let contribution = upcomingContributions[indexPath.row]
+      let publisher = publishers[indexPath.row]
```

Comment:
Why not `guard let publisher = publishers[safe: indexPath.row]` ?

Reply:
No issue adding it, just that since indexPath is tightly bound with publishers list, it will not happen. Also if somehow something does make a change and not reload tableview, in that case it will crash and we can debug. With safe we can either do assertFail or return an empty cell.

WDYT assertFail or return empty cell?

Reply:
You can do both? :)

```swift
assertionFailure("No publisher found for indexPath: \(indexPath)")
return UITableViewCell()
```

Assertions will be compiled out at release and won't crash (although will look funny :P)

Reply:
Cool will add both then :)